src/picasso-components/react-components/data-title-component.jsx

```diff
@@ -244,9 +241,12 @@ export default function createDataTitileComp() {
       if (minWidth) {
         style.minWidth = minWidth;
       }
+
```

Comment:
Could you remove unnecessary new line changes? Line 246 too

Reply:
ok. Removed.